pkg/virt-handler/virt-chroot/virt-chroot.go

```diff
@@ -20,28 +20,82 @@
 package virt_chroot
 
 import (
+	"fmt"
 	"os/exec"
 	"strings"
 
+	linuxproc "github.com/c9s/goprocinfo/linux"
```

Comment:
Could we avoid this dependency? The repository has quite few contributors, however it doesn't seems to be very active. I'd avoid pulling this dependency and reimplement the proc handling here